config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml

```diff
@@ -251,3 +251,42 @@ presubmits:
           env:
             - name: GOPATH
               value: /go
+  - name: pull-kubernetes-node-crio-1.18-e2e
+    skip_branches:
+    - release-\d+\.\d+  # per-release image
+    annotations:
+      fork-per-release: "true"
+      testgrid-alert-stale-results-hours: "24"
+      testgrid-create-test-group: "true"
+      testgrid-num-failures-to-alert: "10"
```

Comment:
I think the bazel tests are failing because of missing testgrid tab:

```
2020/07/22 11:29:37 FAIL: could not write config: 1 error occurred:
	* configuration error for (TestGroup) pull-kubernetes-node-crio-1.18-e2e: Each Test Group must be referenced by at least 1 Dashboard Tab.
```
I believe you will need to update the testgrid config file and add an entry for your new test. Something like what is done here:

https://github.com/kubernetes/test-infra/blob/master/config/testgrids/kubernetes/sig-node/config.yaml#L41